src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -124,4 +125,50 @@ theme.alert.error.background = theme.red;
 //alias warn to warning
 theme.alert.warn = theme.alert.warning;
 
+theme.button = {
```

Comment:
eventually all of these should be linked to actual colors but I 100% don't think you should worry about that now!